integrations/build_system/make.rst

```diff
@@ -3,22 +3,50 @@
 Make
 ====
 
-Conan provides integration with plain Makefiles by means of the ``make`` generator. If you are using ``Makefile`` to build your project you
-could get the information of the dependencies in a *conanbuildinfo.mak* file. All you have to do is indicate the generator like this:
+Conan provides two integrations for plain Makefiles:
+#. The :ref:`Make generator<make_generator>`
+#. The :ref:`Make toolchain<make_toolchain>`
```

Comment:
Please include here too a note that this is experimental and subject to breaking change.